code/datums/gameticker.dm

```diff
@@ -69,7 +71,14 @@ var/global/current_state = GAME_STATE_INVALID
 				// as part of the early start most people might not even see it at 150
 				// so this makes it show up a minute before the game starts
 				handle_mapvote()
-				did_mapvote = 1
+				did_mapvote = TRUE
+
+			if (pregame_timeleft <= 30 && !did_reminder)
+				// hey boo the rounds starting and you didnt ready up
+				for_by_tcl(P, /mob/new_player)
+					if (!P.ready)
+						playsound(P.loc, 'sound/misc/clock_tick.ogg', 50, flags = SOUND_IGNORE_SPACE)
```

Comment:
This will be incredibly loud as the sound will be played N times where N is the number of players not readied up. Please don't do this and instead use `playsound_global` to which you pass a list of mobs or clients as I described in #imcoder.

Reply:
oh god what its gonna play for everyone on there i did not know that
okie on it